app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    for person in people:
+        Person(name=person["name"], age=person["age"])
+
+    for person in people:
+        name = person["name"]
+
+        if person.get("wife"):
+            Person.people[name].wife = Person.people[person["wife"]]
```

Comment:
You could retrieve person instance from the beginning instead of name and operate further with it
```suggestion
        person_instance = Person.people[person["name"]]

        if person.get("wife"):
            person_instance.wife = Person.people[person["wife"]]
```